README.md

```diff
@@ -66,7 +79,7 @@ Conditionally enable/disable the agent. For example, you will likely want to dis
 
 By default, IOpipe will capture timeouts by exiting your function 0.5 seconds early from the AWS configured timeout, to allow time for reporting. You can disable this feature by setting `timeout_window` to `0` in your configuration. If not supplied, the environment variable `$IOPIPE_TIMEOUT_WINDOW` will be used if present.
 
-### Reporting Exceptions
+## Reporting Exceptions
```

Comment:
This should be a `###` as in the table of contents it is on the same level as Configuration